docs/docs-reanimated/docs/device/useAnimatedKeyboard.mdx

```diff
@@ -120,3 +124,72 @@ import AnimatedKeyboardSrc from '!!raw-loader!@site/src/examples/AnimatedKeyboar
 ## Platform compatibility
 
 <PlatformCompatibility android ios />
+
+## Migration guide
+
+As of `react-native-reanimated` **4.x** the `useAnimatedKeyboard` hook has been deprecated in favor of the more robust and feature-complete [keyboard](https://kirillzyusko.github.io/react-native-keyboard-controller/) library. This guide walks you through the migration process.
+
+### Step 1: Install `react-native-keyboard-controller`
+
+Follow the official installation instructions in the [installation guide](https://kirillzyusko.github.io/react-native-keyboard-controller/docs/installation).
+
+### Step 2: Add a compatibility wrapper
+
+To make migration easier, you can drop in a small compatibility layer:
+
+```ts
+import { useKeyboardHandler } from 'react-native-keyboard-controller';
+import { useSharedValue } from 'react-native-reanimated';
+
+export const KeyboardState = {
+  UNKNOWN: 0,
+  OPENING: 1,
+  OPEN: 2,
+  CLOSING: 3,
+  CLOSED: 4,
+};
+
+export const useAnimatedKeyboard = () => {
+  const height = useSharedValue(0);
+  const state = useSharedValue(KeyboardState.UNKNOWN);
+
+  useKeyboardHandler(
+    {
+      onStart: (e) => {
+        'worklet';
+
+        state.set(e.height > 0 ? KeyboardState.OPENING : KeyboardState.CLOSING);
+      },
+      onMove: (e) => {
+        'worklet';
+
+        height.set(e.height);
+      },
+      onEnd: (e) => {
+        'worklet';
+
+        state.set(e.height > 0 ? KeyboardState.OPEN : KeyboardState.CLOSED);
+        height.set(e.height);
+      },
+    },
+    []
+  );
+
+  return { height, state };
+};
```

Comment:
@kirillzyusko Would it be possible for you to add this compatibility hook in your library to make the migration process easier? This way the developers would only need to change the import, i.e.:
```diff
-import { useAnimatedKeyboard } from 'react-native-reanimated';
+import { useAnimatedKeyboard } from 'react-native-keyboard-controller';
```